docs/creation/supply-chain.md

```diff
@@ -46,7 +48,7 @@ actor TempWriter
 We're now left with one more compiler error to address.
 
 ```text
-x.pony:13:10: couldn't find write in None val
+x.pony:14:10: couldn't find write in None val
```

Comment:
It will not let me comment on the exact line so leaving one here just above it.

> One more change address that `file` could be uninitialized:

Should be

> One more change addresses that `_file` could be uninitialized: